sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/eventsourcedentity/EventSourcedEntity.java

```diff
@@ -31,6 +31,25 @@
 import java.util.function.Function;
 
 /**
+ * The Event Sourced state model captures changes to data by storing events in a journal.
+ * The current entity state is derived from the emitted events.
+ * <p>
+ * When implementing an Event Sourced Entity, you first define what will be its internal state (your domain model),
+ * the commands it will handle (mutation requests) and the events it will emit (state changes).
```

Comment:
"acceptable requests"? instead of "mutation requests"

Reply:
Agree, we can also have read-only commands. 

Not sure about "acceptable requests" though. I will try to improve it.

Reply:
I'm replacing it by: 

> the commands it will handle and the events it will emit to modify its state

So not being explicit about what a command handler is. It can be a read-only or a command that emits events.

Reply:
yeah, simple and expressive.